src/GmailConnection.php

```diff
@@ -168,7 +168,13 @@ public function setBothAccessToken( $token )
 	public function saveAccessToken( array $config )
 	{
 		$fileName = $this->getFileName();
-		$file = "gmail/tokens/$fileName.json";
+		$user = auth()->user();
+
+        	if ($user) {
```

Comment:
I would add a config variable so they can enable the multiple credentials because.. I can give you an example:
If you have multiple users but you want to use the same Gmail account for all of them, this wouldn't allow that.
And if they want to use multiple users, they just have to change the config/.env variable to `true`.